spring-boot-web-application/src/main/java/com/in28minutes/springboot/service/StudentService.java

```diff
@@ -24,6 +24,11 @@ public class StudentService {
                 "Hiker, Programmer and Architect", new ArrayList<>(List.of(courseOne, courseTwo)));
 
         students.add(ranga);
+
+        Student slava = new Student("Student2", "Slava Lapitski",
+                "Hiker, Programmer and Architect", new ArrayList<>(List.of(courseOne, courseTwo)));
+
+        students.add(ranga);
     }
 
     public Student retrieveStudent(String studentId) {
```

Comment:
1) Duplicate student addition: You are adding the same student 'ranga' twice to the students list. Replace the second 'students.add(ranga);' with 'students.add(slava);' to add the new student 'slava' instead.